src/Discord.Net.Core/Entities/Channels/Direction.cs

```diff
@@ -3,6 +3,14 @@ namespace Discord
     /// <summary>
     ///     Specifies the direction of where message(s) should be gotten from.
     /// </summary>
+    /// <remarks>
+    ///     This enum is used to specify what direction should the message retrieval go.
```

Comment:
```suggestion
    ///     This enum is used to specify the direction for retrieving messages.
```
Not sure if this sounds too similar to the summary now.